compliance_checker/suite.py

```diff
@@ -147,7 +153,8 @@ def _load_checkers(cls, checkers):
 
         for c in checkers:
             try:
-                check_obj = c.resolve()
+                # check_obj = c.resolve()
+                check_obj = c.load()
```

Comment:
I don't follow this change. Is it related with `pkg_resources`?

Reply:
yes, the pkg_resources `EntryPoint` object has a .`resolve` method, and the importlib one has `load()` instead. I"m not totally sure they are exactly the same, but this seemed to work.